sly.el

```diff
@@ -2707,7 +2720,7 @@ Debugged requests are ignored."
 ;;;;; Event logging to *sly-events*
 ;;;
 ;;; The *sly-events* buffer logs all protocol messages for debugging
-;;; purposes. 
+;;; purposes.
```

Comment:
Nitpick: this whitespace change shouldn't be here should it?  (it's fine to submit these in a separate PR, if you want)